### PR TITLE
Added Microsoft.Bicep v0.4.412

### PR DESCRIPTION
**Pull request by dulfer**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/21191)

### CODE REVIEW

File: manifests/m/Microsoft/Bicep/0.4.412/Microsoft.Bicep.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+- azure
+- bicep
+- bicep cli
+ShortDescription: Bicep is a Domain Specific Language (DSL) for deploying Azure resources declaratively. It aims to drastically simplify the authoring experience with a cleaner syntax and better support for modularity and code re-use.
+PackageUrl: https://github.com/Azure/bicep
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/Azure/bicep/releases/download/v0.4.412/bicep-setup-win-x64.exe
+  InstallerSha256: 4595e286d2b78d922ebc9769a4c206cf2063f1c6f0339a52730904a6a335494e
+  InstallerType: exe
```
Comment (quhxl) [inline manifests/m/Microsoft/Bicep/0.4.412/Microsoft.Bicep.yaml:19]:
```suggestion
  InstallerType: inno
```

File: manifests/m/Microsoft/Bicep/0.4.412/Microsoft.Bicep.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+PackageIdentifier: Microsoft.Bicep
+PackageVersion: 0.4.412
```
Comment (quhxl) [inline manifests/m/Microsoft/Bicep/0.4.412/Microsoft.Bicep.yaml:2]:
```suggestion
PackageVersion: 0.4.412.5873
```

File: manifests/m/Microsoft/Bicep/0.4.412/Microsoft.Bicep.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+ShortDescription: Bicep is a Domain Specific Language (DSL) for deploying Azure resources declaratively. It aims to drastically simplify the authoring experience with a cleaner syntax and better support for modularity and code re-use.
+PackageUrl: https://github.com/Azure/bicep
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/Azure/bicep/releases/download/v0.4.412/bicep-setup-win-x64.exe
+  InstallerSha256: 4595e286d2b78d922ebc9769a4c206cf2063f1c6f0339a52730904a6a335494e
+  InstallerType: exe
+  InstallerSwitches:
+    Silent: /VERYSILENT /SUPPRESSMSGBOXES /NORESTART
+    SilentWithProgress: /SILENT /SUPPRESSMSGBOXES /NORESTART
```
Comment (quhxl) [inline manifests/m/Microsoft/Bicep/0.4.412/Microsoft.Bicep.yaml:22]:
```suggestion
```

**Review by quhxl — CHANGES_REQUESTED** [review]
Make sure to rename your folder. (e.g. in Github Desktop)

**Comment by quhxl** [comment]
Please add a comment to close #21189.

File: manifests/m/Microsoft/Bicep/0.4.412/Microsoft.Bicep.yaml (hunk 2 above)
Comment (quhxl) [reply manifests/m/Microsoft/Bicep/0.4.412/Microsoft.Bicep.yaml:2]: The `PackageVersion` has to match the entry in the Add/Remove Programs list.

File: manifests/m/Microsoft/Bicep/0.4.412/Microsoft.Bicep.yaml (hunk 3 above)
Comment (quhxl) [reply manifests/m/Microsoft/Bicep/0.4.412/Microsoft.Bicep.yaml:22]: These switches aren't necessary bc the cli knows how to handle inno installers.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Version folder needs to be updated.

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**